app/src/main/java/edu/mit/mitmobile2/dining/activities/DinningHouseActivity.java

```diff
@@ -0,0 +1,104 @@
+package edu.mit.mitmobile2.dining.activities;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.v4.view.ViewPager;
+import android.view.Menu;
+import android.view.MenuItem;
+import android.view.View;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import com.squareup.picasso.Picasso;
+
+import java.text.SimpleDateFormat;
+import java.util.Calendar;
+
+import butterknife.ButterKnife;
+import butterknife.InjectView;
+import butterknife.OnClick;
+import edu.mit.mitmobile2.Constants;
+import edu.mit.mitmobile2.MITActivity;
+import edu.mit.mitmobile2.R;
+import edu.mit.mitmobile2.dining.adapters.HouseMenuPagerAdapter;
+import edu.mit.mitmobile2.dining.model.MITDiningHouseVenue;
+
+public class DinningHouseActivity extends MITActivity {
+
+    @InjectView(R.id.house_image_view)
+    ImageView houseImageView;
+    @InjectView(R.id.house_name_text_view)
+    TextView houseNameTextView;
+    @InjectView(R.id.house_hours_text_view)
+    TextView houseHoursTextView;
+    @InjectView(R.id.dinning_house_menu_viewpager)
+    ViewPager houseMenuViewpager;
+    @InjectView(R.id.date_text_text_view)
+    TextView dateTextView;
+    @InjectView(R.id.info_text_view)
+    TextView infoTextView;
+    @InjectView(R.id.forward_image_view)
+    ImageView forwardImageView;
+    @InjectView(R.id.back_image_view)
+    ImageView backImageView;
+
+    @OnClick(R.id.info_image_view)
+    public void gotoHouseInfo() {
+        Intent intent = new Intent(getApplicationContext(), DinningHouseInfoActivity.class);
```

Comment:
No need for ApplicationContext here, use "this" for the context of the activity itself